libs/create_library_skeleton.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+'''
+Copyright (c) 2018 Thomas Heller
+
+Distributed under the Boost Software License, Version 1.0. (See accompanying
+file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+create_library_skeleton.py - A tool to generate a library skeleton to be used
+as library component of HPX
+'''
+
+import sys, os
+
+if len(sys.argv) != 2:
+    print('Usage: %s <lib_name>' % sys.argv[0])
+    print('Generates the skeleton for lib_name in the current working directory')
+    sys.exit(1)
+
+lib_name = sys.argv[1]
+
+# CMake minimum version
+cmake_version = '3.3.2'
+
+cmake_header =  '# Copyright (c) 2019 The STE||AR-Group\n'
```

Comment:
Wouldn't python's multiline strings work here? And down below in the calls to `f.write`. Just for readability...

Reply:
I'd even suggest separating the template text into their own files and letting the script fill in the blanks. Python is good at that kind of work. That will simplify maintenance.